driver.go

```diff
@@ -257,10 +258,114 @@ func findNeighbors(tx neo4j.Transaction, ids []uint64) ([]neo4jNode, []neo4jNode
 	return sources, targets, edges, nil
 }
 
-func MakeProperties(input map[string]interface{}) map[string]*ls.PropertyValue {
+func MakePropertiesForSetNodeValue(cfg Config, node graph.Node, input map[string]interface{}) interface{} {
+	// check if value type exists in config
+	for k, v := range input {
```

Comment:
Instead of for loop, simply do: v, ok:=input[cfg.Map(ls.NodeValueTerm)]
Call neo4jValueToNativeValue
And call SetNodeValue()
Remove rest of the func